### ISSUE TITLE
How to create  Property Names and IDs  on  regtest network

### ISSUE THREAD

**Issue by XieWeiZM** [OmniLayer/omnicore#591]
How to create  Property Names and IDs  on  regtest network

**Comment by XieWeiZM** [comment]
i use omni_sendissuancefixed how to appoint this propertyid

**Comment by hhy464591454** [comment]
@XieWeiZM  I have also encountered this problem. If you want to solve it, please let me know. Thank you.

**Comment by dexX7** [comment]
Hey guys,

propertyids are allocated by the protocol when the creation transaction confirms.

You can use `omni_sendissuancefixed` (or one of the others) to create a token, store the txid and then use `omni_gettransaction txid`. The result has a field with `propertyid`.

**Comment by XieWeiZM** [comment]
@hhy464591454  @dexX7   That is to say I can't specify propertyids ！！！！oh no ~~~~~~

**Comment by dexX7** [comment]
Right, you can't: they are assigned in chronological order by the protocol.

May I ask why you need to specify them explicitly?

**Comment by dexX7** [comment]
And you're working with hardcoded property ids?

Hmm.. how about creating some junk properties so your Test-USDT gets the identifier 31?

**Comment by houjingwei** [comment]
@dexX7 
您可以使用omni_sendissuancefixed（或其他人之一）创建令牌，存储txid然后使用omni_gettransaction txid。结果有一个领域propertyid。

结果里边没有propertyid呀
我的结果是
{
  "txid": "6ad67f562734bb23392fcf7efdc4e93e0e45c237c180cb4766d8ea8cd9e2dc24",
  "fee": "0.00005500",
  "sendingaddress": "mo4Xz2wSiAb2eTtxg4FUrj7qb6bU3iL22i",
  "ismine": true,
  "version": 0,
  "type_int": 50,
  "type": "Create Property - Fixed",
  "ecosystem": "test",
  "propertytype": "indivisible",
  "category": "leleCate",
  "subcategory": "leleSubCate",
  "propertyname": "leleToken",
  "data": "lele-data",
  "url": "lele-url",
  "amount": "100000000",
  "confirmations": 0
}

**Comment by dexX7** [comment]
Wait until confirmation, then the property id is shown.

**Comment by XieWeiZM** [comment]
@dexX7 I can configure propertyid，is  propertyid  2   and propertyid  31  similar？？i can test USDT use Test Omni'propertyid  ??

**Comment by dexX7** [comment]
Yes, you can.

**Comment by XieWeiZM** [comment]
@dexX7  Thanks

**Comment by SftJun** [comment]
@houjingwei @dexX7 
&emsp;&emsp;He gays, i setup omnicore on docker. When get property of usdt id:[`2147484026`] failed.
### This is error
`omni_getproperty 2147484026 # 2147484026: usdt property id on testnet`
 ```
{
    "result":null,
    "error":{
        "code":-8,
        "message":"Property identifier does not exist"
    },
    "id":"1"
}
```
### This is successed.
`omni_listproperties #command `

```
{
    "result":[
        {
            "propertyid":1,
            "name":"Omni",
            "category":"N/A",
            "subcategory":"N/A",
            "data":"Omni serve as the binding between Bitcoin, smart properties and contracts created on the Omni Layer.",
            "url":"http://www.omnilayer.org",
            "divisible":true
        },
        {
            "propertyid":2,
            "name":"Test Omni",
            "category":"N/A",
            "subcategory":"N/A",
            "data":"Test Omni serve as the binding between Bitcoin, smart properties and contracts created on the Omni Layer.",
            "url":"http://www.omnilayer.org",
            "divisible":true
        }
    ],
    "error":null,
    "id":"1"
}
```
### **May i ask how to get right property id? thanks very much!!!**

**Comment by kazaff** [comment]
@dexX7  hi~

can i have the detail of `omni_sendissuancefixed` to use?

I make the rpc request like the [doc](https://github.com/OmniLayer/omnicore/blob/master/src/omnicore/doc/rpc-api.md#omni_sendissuancefixed) said:
```
omnicore-cli "omni_sendissuancefixed" \
    "my address who owned enough tbtc coin" 2 1 0 "Companies" "Bitcoin Mining" \
    "Quantum Miner" "" "" "1000000"
```
But i got the error back. 

I found that the doc described that every parameters have been required, so why the demo DO NOT set the last three params? And how to set them? What do they means?

thanks in advance~

**Comment by dexX7** [comment]
Hi @kazaff,

actually it looks like the documentation isn't correct for this call, because it shows the fields for crowdsales.

Which error do you get?

**Comment by guoxingyun** [comment]
When I use the omni_gettransaction query ,I get
amount": "1000000",
  "valid": false,
  "invalidreason": "Unknown error",

than,The log shows this information

2018-11-12 08:13:12 ERROR: Entry (63964e618244637584973e3922b3d8bf5dcea15e729fdea0b5e1c614e9a573a5) could not be loaded from OmniTXDB!
2018-11-12 08:13:12 findSPByTX(): ERROR: failed to find property created with 63964e618244637584973e3922b3d8bf5dcea15e729fdea0b5e1c614e9a573a5ERROR: Entry (63964e618244637584973e3922b3d8bf5dcea15e729fdea0b5e1c614e9a573a5) could not be loaded from OmniTXDB!
